### ISSUE TITLE
To center pages

### ISSUE THREAD

**Issue by EvelinaBiserova** [ugeneunipro/ugeneunipro.github.io#36]
Modern sites usually place information on the pages in the center, but our content is pressed to the left (with the exception of the "Contacts" page)

**Comment by yalgaer** [comment]
I agree this improvement should be done once all bugs are fixed after the launch.
We should also consider to improve a usability on touch devices: our site is not ready for mobiles: layout is not responsive and hover-powered menus are not compatible with touch events.
